app/javascript/utils/form/final_form/Field.js

```diff
@@ -51,7 +51,7 @@ export const TextField = ({ fieldName, label, blockNote, validation, placeholder
             input={input}
             meta={meta}
             id={id || `form-${fieldName}`}
-            type='text'
+            type={type || 'text'}
```

Comment:
what other type could it be? it's called TextField

Reply:
it can be also a `number`

Reply:
hrm. were there still plans to do a bigger refactor of all of these fields in this PR? I'm wondering if we can consolidate some of these types and make them a bit clearer, or find a clearer name for what this component is.

Can we combine this field with InputField? and then pass in number vs text and whether or not to include a label?

Reply:
Maybe we can pair on this one? I've already cleaned up those fields a little bit in the previous PR. Renaming this one one to `InputField` would make sense, but there is already `InputField`, that handles just an input. This one is more of a input group, with label, errors etc. Label is optional though.